src/runtime/inspector_client.cpp

```diff
@@ -93,6 +100,12 @@ void TrInspectorClient::tick()
 {
   recv(); // recv firstly, then parse
 
+  if (connectionType_ == ConnectionType::WEBSOCKET)
```

Comment:
Use if else instead of the if-return:

```cpp
if (connectionType_ == ConnectionType::WEBSOCKET)
{
   // handle websocket frame
}
else if (connectionType_ == ConnectionType::HTTP)
{
  // handle http
}
else
{
  assert(false && "....");
}
```